### ISSUE TITLE
Will CTPN work with Tensorflow 1.5.0 and cuda 9?

### ISSUE THREAD

**Issue by VishnuPrasadJ** [eragonruan/text-detection-ctpn#137]
I am currently using Tensorflow 1.3.0 and cuda 8 and CTPN is working just fine.I need to know will CTPN work on Tensorflow 1.5.0 and cuda 9.

**Comment by SophieSongSong** [comment]
yes. I have done this before. but in my exprience, cuda8 works faster than cuda9 on text-detection-ctpn.
